test/integrations/destinations/criteo_audience/network.ts

```diff
@@ -1,3 +1,23 @@
+const headers = {
```

Comment:
This file should only contain business scenario related mocks. 

`Oauth` and `Other` scenarios should not be added here. Also each mock should have a description